### PR TITLE
Migration to webpack 2

### PR DESCRIPTION
**Pull request by pirelenito**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -335,7 +335,6 @@ Possible values:
 - `font`
 - `image`
 - `javaScript`
-- `json`
```
Comment (xaviervia) [inline README.md]: Why should this be removed?

File: src/configure-webpack/index.js [hunk 2]
```diff
@@ -71,13 +72,6 @@ const buildSharedWebpackConfig = (saguiConfig) => {
       // To ensure hashes are generated based on the file contents, use webpack-md5-hash plugin.
       ...(action === actions.BUILD ? [new WebpackMd5Hash()] : []),
 
-      // Use "OccurrenceOrderPlugin" in order to make build deterministic.
```
Comment (xaviervia) [inline src/configure-webpack/index.js]: I’m assuming this are Webpack defaults now.

File: src/configure-webpack/index.js [hunk 3]
```diff
@@ -106,7 +99,7 @@ const buildSharedWebpackConfig = (saguiConfig) => {
       // Should first try to resolve loaders nested within Sagui.
       // This fixes an issue in NPM v2 where webpack incorrectly
       // thinks that the package `eslint` is the `eslint-loader`
-      modulesDirectories: [
```
Comment (xaviervia) [inline src/configure-webpack/index.js]: The comments points out to an npm 2 problem, but I imagine npm 2 is _super deprecated_ at this point. Is this in general necessary?

File: src/configure-webpack/loaders/font.js [hunk 4]
```diff
@@ -5,26 +5,64 @@ export default {
...
       module: {
-        loaders: [
+        rules: [
           {
             test: fileExtensions.test.WOFF,
-            loader: 'file-loader?name=[name]-[hash].[ext]&mimetype=application/font-woff'
+            use: [
+              {
+                loader: 'file-loader',
+                options: {
```
Comment (xaviervia) [inline src/configure-webpack/loaders/font.js]: Such readability, wow.

File: README.md (hunk 1 above)
Comment (pirelenito) [inline README.md]: It is no longer needed in webpack 2. They do it by default.

File: src/configure-webpack/index.js (hunk 2 above)
Comment (pirelenito) [inline src/configure-webpack/index.js]: Exactly!

File: src/configure-webpack/loaders/font.js (hunk 4 above)
Comment (pirelenito) [inline src/configure-webpack/loaders/font.js]: hehe

File: src/configure-webpack/index.js (hunk 3 above)
Comment (pirelenito) [inline src/configure-webpack/index.js]: Good point, maybe we can update it. I think it is still a safe bet to be explicit about where to look for loaders.

Comment (xaviervia) [inline src/configure-webpack/index.js]: Fair enough, just updating the comment for now should do.

**Pull request merged**